index.js

```diff
@@ -205,7 +216,8 @@ async function run() {
     }
 
     for (const destination of config.DESTINATIONS) {
-        await attemptPaymentToDestination({ destination, outChannelIds }).catch(err => {
+        const first_attempt = 0
+        await attemptPaymentToDestination({ destination, outChannelIds, first_attempt}).catch(err => {
```

Comment:
`first_attempt` is not needed here. Just pass `0` to the function as an argument?!
```javascript
await attemptPaymentToDestination({ destination, outChannelIds, 0 }).catch(err => {
```